src/fu-tool.c

```diff
@@ -2569,6 +2569,11 @@ fu_util_esp_list (FuUtilPrivate *priv, gchar **values, GError **error)
 	return TRUE;
 }
 
+static gboolean
+_g_str_equal0 (gconstpointer str1, gconstpointer str2)
+{
+	return g_strcmp0 (str1, str2) == 0;
+}
```

Comment:
to save a few lines of duplication maybe this shouldn't be static and in `fu-util-common.c`?

Reply:
I'm lobbying to get it in Glib. :)